### PR TITLE
Chore/style member invites component

### LINKED ISSUE #70: style member invites component

### PR DESCRIPTION
**Pull request by gregorykan**
closes https://github.com/root-systems/cobuy/issues/70

### CODE REVIEW

File: agents/components/MemberInvites.js [hunk 1]
```diff
@@ -81,17 +85,25 @@ function MemberInvites (props) {
...
-          }
-      />
+      <p className={styles.intro}>
+        <FormattedMessage
+          id='agents.memberInvites'
+          className={styles.labelText}
+        />
+      </p>
+      <div className={styles.groupNameContainer}>
+        <Field
```
Comment (ahdinosaur) [inline agents/components/MemberInvites.js]: hey, i don't think we should have group name as part of the member invites component, this is handled by the group agent profile component, right?

**Review by ahdinosaur — COMMENTED** [review]
nice!

two main comments:

- it's not responsive (see screenshot in next comment)
- i don't think we should ask the group name here as i expect we use the profile component to edit the group name, etc

**Comment by ahdinosaur** [comment]
not responsive:

cobuy-member-invites

should it be?

**Comment by ahdinosaur** [comment]
screenshot:

cobuy-member-invites1

@gregorykan would be appreciated if you could post screenshots or gifs as part of the pull request, since it helps document our progress for stakeholders and future archaeologists (like ourselves). for gifs i use https://github.com/phw/peek. :smile:

File: agents/components/MemberInvites.js (hunk 1 above)
Comment (gregorykan) [reply agents/components/MemberInvites.js]: @ahdinosaur good point ! will remove

**Comment by gregorykan** [comment]
responsive-member-invites

**Pull request merged**